node/Cargo.toml

```diff
@@ -27,6 +30,7 @@ serde = { version = "1", features = ["derive"] }
 toml = "0.5"
 directories = "4.0"
 paste = "1.0"
+thiserror = "1.0"
```

Comment:
Well... we do have anyhow already... even `Box<dyn Error>` isn't necessary, but I'm OK with it. Not sure why we need thiserror.